org-code-javabuilder/lib/src/test/java/org/code/javabuilder/InputRedirectionStreamTest.java

```diff
@@ -4,6 +4,7 @@
 import static org.mockito.Mockito.*;
 
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
```

Comment:
All changes in this file were unrelated to this change. They are test fixes that are required for tests to pass on Windows due to the different line separator used on Windows.

Reply:
Ah interesting! Separate discussion, but should we not be using `System.lineSeparator()` then?